Library/Homebrew/formula.rb

```diff
@@ -1043,7 +1043,7 @@ def systemd_timer_path
   def service
     return unless service?
 
-    Homebrew::Service.new(self, &self.class.service)
+    @service ||= Homebrew::Service.new(self, &self.class.service)
```

Comment:
Was there any special reason why this wasn't memoized before?

Reply:
Probably not.